### PR TITLE
Disambiguate name conflicts with formatted names

### PR DESCRIPTION
**Pull request by fearthecowboy**

### CODE REVIEW

File: modelerfour/flattener/flattener.ts [hunk 1]
```diff
@@ -70,6 +70,7 @@ export class Flattener {
         request.parameters?.push(vp);
       }
     }
+
```
Comment (daviwil) [inline modelerfour/flattener/flattener.ts:73]: Unneeded?

File: modelerfour/flattener/flattener.ts [hunk 2]
```diff
@@ -194,8 +195,7 @@ export class Flattener {
       for (const group of this.codeModel.operationGroups) {
         for (const operation of group.operations) {
           // when there are multiple requests in an operation
-          // and the 
-          // skip flattening of requests when  
+          // and the geneator asks not to flatten them
```
Comment (daviwil) [inline modelerfour/flattener/flattener.ts]:
```suggestion
          // and the generator asks not to flatten them
```

**Review by daviwil — APPROVED** [review]
Looks like the regression is the same one we saw from the last merge, nothing new for this change.

**Comment by daviwil** [comment]
By the way, did any of the scenario YAML files get updated after this change?

**Comment by fearthecowboy** [comment]
Whoops forgot to push after the commit with the scenario regen

**Comment by fearthecowboy** [comment]
oh, no, no changes in scenario files. that's why.

**Pull request merged**